python/rpdk/typescript/templates/handlers.ts

```diff
@@ -11,7 +11,7 @@ import {
     ResourceHandlerRequest,
     SessionProxy,
 } from '{{lib_name}}';
-import { ResourceModel } from './models';
+import { ResourceModel, TypeConfigurationModel } from './models';
```

Comment:
`TypeConfigurationModel` is coming from `./models` component but could not find it in the change. Is it missing?